.gitignore

```diff
@@ -1 +1,3 @@
 # Please add IDE and OS specific files in a global gitignore on your system
+
+node_modules
```

Comment:
might want to add https://www.toptal.com/developers/gitignore/api/node. I don't want to have people keep committing these files accidentally